<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="gatewayNameLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Nouveau Nom Paserelle :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="gatewayNameEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AppliquerModification">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Appliquer la configuration</string>
    </property>
   </widget>
   <widget class="QLabel" name="ipAddressLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Nouvelle Adresse Ip :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipAddressEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="modifyGatewayButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifer Passerelle M50</string>
    </property>
   </widget>
   <widget class="QPushButton" name="acquisitionModelButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifier modele d'aquisition I-30</string>
    </property>
   </widget>
   <widget class="QPushButton" name="measureModelButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>110</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifer modele de mesure U-10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Retour</string>
    </property>
   </widget>
   <widget class="QLabel" name="ipAddressPasserelleLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Nouvelle Adresse Ip de la passerelle:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ipAddressMaskLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Nouveau Mask de sous réseaux :</string>
    </property>
   </widget>
   <widget class="QLabel" name="ipAddressDNSLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Nouvelle Adresse de serveur DNS :</string>
    </property>
   </widget>
   <widget class="QLabel" name="TimeZoneLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Time zone de la passerelle :</string>
    </property>
   </widget>
   <widget class="QLabel" name="DHCPLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>DHCP :</string>
    </property>
   </widget>
   <widget class="QLabel" name="EtatModbusLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Modbus :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipAddressPasserelleEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipAddressMaskEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>280</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipAddressDNSEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>340</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxDHCP">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>470</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxModbus">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>520</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxTimeZone">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>410</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="NameModeleAquisitionLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Nouveau nom modele acquisition  :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NameModeleAcquisitionEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="returnButton2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Retour</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnButton3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Retour</string>
    </property>
   </widget>
   <widget class="QLabel" name="PeriodeIntegrationInst">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Valeur de periode d'integration instantaner (*0.2s) :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PeriodeIntegrationInstEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="status1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Status du Capteur N°1 : </string>
    </property>
   </widget>
   <widget class="QLabel" name="status2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Status du Capteur N°2 : </string>
    </property>
   </widget>
   <widget class="QLabel" name="status3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Status du Capteur N°3 : </string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxStatus1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxStatus2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>280</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxStatus3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>350</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="TypeCharge">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Type de charge :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxCharge">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="PeriodeIntegrationMoy">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Valeur de periode de mesure moyen (en minute) :</string>
    </property>
   </widget>
   <widget class="QLabel" name="PeriodeIntegrationCourbe">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Valeur de periode de mise a jour de courbe des charges (en minute) :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PeriodeIntegrationMoyEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>170</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="PeriodeIntegrationCourbeEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>230</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="NameDomaine">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>570</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom Domaine :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NameDomaineEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>580</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="PrimaryKeylabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Configuration Dispositif :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="PrimaryKey">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="idClientEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="idClientLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Nouvelle id Client :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnButton4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Retour</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClientWebButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifier Client Web</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="MDPEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="MDPLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Nouveau Mot de passe :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AppliquerModification_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Appliquer la configuration</string>
    </property>
   </widget>
   <zorder>status1</zorder>
   <zorder>PeriodeIntegrationMoy</zorder>
   <zorder>status2</zorder>
   <zorder>status3</zorder>
   <zorder>comboBoxStatus3</zorder>
   <zorder>comboBoxStatus2</zorder>
   <zorder>PeriodeIntegrationCourbe</zorder>
   <zorder>comboBoxStatus1</zorder>
   <zorder>comboBoxCharge</zorder>
   <zorder>AppliquerModification</zorder>
   <zorder>ipAddressLabel</zorder>
   <zorder>ipAddressEdit</zorder>
   <zorder>returnButton</zorder>
   <zorder>ipAddressPasserelleLabel</zorder>
   <zorder>ipAddressMaskLabel</zorder>
   <zorder>ipAddressDNSLabel</zorder>
   <zorder>TimeZoneLabel</zorder>
   <zorder>DHCPLabel</zorder>
   <zorder>EtatModbusLabel</zorder>
   <zorder>ipAddressPasserelleEdit</zorder>
   <zorder>ipAddressMaskEdit</zorder>
   <zorder>ipAddressDNSEdit</zorder>
   <zorder>comboBoxDHCP</zorder>
   <zorder>comboBoxModbus</zorder>
   <zorder>comboBoxTimeZone</zorder>
   <zorder>NameModeleAquisitionLabel</zorder>
   <zorder>NameModeleAcquisitionEdit</zorder>
   <zorder>returnButton2</zorder>
   <zorder>returnButton3</zorder>
   <zorder>PeriodeIntegrationInst</zorder>
   <zorder>PeriodeIntegrationInstEdit</zorder>
   <zorder>TypeCharge</zorder>
   <zorder>PeriodeIntegrationMoyEdit</zorder>
   <zorder>PeriodeIntegrationCourbeEdit</zorder>
   <zorder>NameDomaine</zorder>
   <zorder>NameDomaineEdit</zorder>
   <zorder>PrimaryKeylabel</zorder>
   <zorder>PrimaryKey</zorder>
   <zorder>gatewayNameEdit</zorder>
   <zorder>idClientEdit</zorder>
   <zorder>idClientLabel</zorder>
   <zorder>gatewayNameLabel</zorder>
   <zorder>returnButton4</zorder>
   <zorder>ClientWebButton</zorder>
   <zorder>MDPEdit</zorder>
   <zorder>MDPLabel</zorder>
   <zorder>AppliquerModification_2</zorder>
   <zorder>modifyGatewayButton</zorder>
   <zorder>acquisitionModelButton</zorder>
   <zorder>measureModelButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>635</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
